### ISSUE TITLE
CSS @media in HTML file breaks syntax highlighting

### ISSUE THREAD

**Issue by aeschli** [atom/language-css#107]
_From @ggregoire on March 10, 2017 17:4_

- VSCode Version: last one
- OS Version: macOS

Hi,

`@media` into `<style></style>` breaks the syntax highlighting.

![screen shot 2017-03-10 at 11 00 17](https://cloud.githubusercontent.com/assets/591804/23804767/e70f76b0-0580-11e7-9340-b00cfc0260f7.png)

_Copied from original issue: Microsoft/vscode#22404_

**Comment by aeschli** [comment]
Reproduce with
```html
<html>
<style>
h1 {
  color: red;
  
}@media only screen {
  
}
</style>
</html>
```

Problem is the lack of a whitespace before `@media`

**Comment by Alhadis** [comment]
Crap, sorry. For some reason, `}@atrule` never occurred to me when writing these rules. :(

Will submit a PR to fix this in a minute.

**Comment by Alhadis** [comment]
FYI, this has absolutely nothing to do with the fact it's embedded inside an HTML document. The same bug is visible in plain CSS, too. `@media` isn't the only rule that's affected, either.

Addressed in #108.
